.changeset/wild-deers-leave.md

```diff
@@ -0,0 +1,5 @@
+---
+"@preact/signals-react": minor
```

Comment:
Let's make this a `patch` change since this a bug fix with no new feature introductions